main.go

```diff
@@ -155,6 +155,13 @@ func loadConfigs() (config.Group, error) {
 		}
 	}
 
+	// Command line Arg
+	if flag.NArg() > 0 {
+		confString := strings.Join(flag.Args(), "\n")
+		confBody := ":" + config.Port + "\n" + strings.Replace(confString, ",", "\n", -1)
```

Comment:
Yes, I understand that, but I'm making a different comparison. :)

```
$ caddy browse gzip
```

I understand that the Caddyfile does not treat spaces as significant, which is why this line `confString := strings.Join(flag.Args(), "\n")` effectively converts the Arg input into Caddyfile format.

With the above example, `confString` becomes:

```
browse
gzip
```